src/main/java/command/Delete.java

```diff
@@ -0,0 +1,37 @@
+package command;
+
+import io.Storage;
+import io.Ui;
+import task.TaskList;
+import util.Parser;
+
+public class Delete implements Command {
+    private final int taskNum;
+
+    private static final String ERROR = "Task not in list.";
```

Comment:
Perhaps the static attributes should be above the private final attributes according to coding standards